### PR TITLE
Show "Pricings" for first activation

### PR DESCRIPTION
**Pull request by purinkle**
Before, we showed people without a registered card a form without any explanation. We felt this was confusing. Updated the activation button to redirect people to "Pricings" for more information.

### CODE REVIEW

File: app/assets/javascripts/components/upgrade_subscription_link.jsx [hunk 1]
```diff
@@ -1,18 +1,63 @@
 import * as Ajax from '../lib/ajax.jsx';
 
 class UpgradeSubscriptionLink extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.checkout = this.checkout.bind(this);
```
Comment (BlakeWilliams) [inline app/assets/javascripts/components/upgrade_subscription_link.jsx]: Not sure about this project, but we don't usually reassign `.bind` functions in the constructor in React components.

We usually call fat arrow functions where we need them. eg: `onPress={() => this.checkout()}`

File: app/assets/javascripts/components/upgrade_subscription_link.jsx [hunk 2]
```diff
@@ -1,18 +1,63 @@
...
+      key: stripePublishableKey,
+      image: iconPath,
+      token: this.upgradeWithNewCard,
+    })
+  }
 
+  showCreditCardForm() {
+    const { userEmailAddress } = Hound.settings;
+    const { nextTier } = this.props;
+    const { price, title } = nextTier;
```
Comment (BlakeWilliams) [inline app/assets/javascripts/components/upgrade_subscription_link.jsx]: `const { price, title } = this.props.nextTier` unless you need `nextTier` maybe?

File: app/assets/javascripts/components/upgrade_subscription_link.jsx (hunk 1 above)
Comment (purinkle) [inline app/assets/javascripts/components/upgrade_subscription_link.jsx]: Resolved in b4e9c8a

File: app/assets/javascripts/components/upgrade_subscription_link.jsx (hunk 2 above)
Comment (purinkle) [inline app/assets/javascripts/components/upgrade_subscription_link.jsx]: Resolved in b4e9c8a

**Pull request merged**